ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -34,6 +34,23 @@ public static bool UniversesAreEqual(Universe universe1, Universe universe2)
             return true;
         }
 
+        public static bool ListsOfCellsAreEqual(List<Cell> cells1, List<Cell> cells2)

```

Comment:
probably not required - see my comment above

Reply:
side note: should equality be reliant on the ordering of the items in the list? compare this to your implementation in `UniversesAreEqual`